drivers/display/display_mcux_dcnano_lcdif.c

```diff
@@ -295,13 +298,24 @@ static const struct display_driver_api mcux_dcnano_lcdif_api = {
 		.dpi_config = {							\
 			.panelWidth = DT_INST_PROP(n, width),			\
 			.panelHeight = DT_INST_PROP(n, height),			\
-			.hsw = DT_INST_PROP(n, hsync),				\
-			.hfp = DT_INST_PROP(n, hfp),				\
-			.hbp = DT_INST_PROP(n, hbp),				\
-			.vsw = DT_INST_PROP(n, vsync),				\
-			.vfp = DT_INST_PROP(n, vfp),				\
-			.vbp = DT_INST_PROP(n, vbp),				\
-			.polarityFlags = DT_INST_PROP(n, polarity),		\
+			.hsw = DT_INST_PROP(n, hsync_len),			\
+			.hfp = DT_INST_PROP(n, hfront_porch),			\
+			.hbp = DT_INST_PROP(n, hback_porch),			\
+			.vsw = DT_INST_PROP(n, vsync_len),			\
+			.vfp = DT_INST_PROP(n, vfront_porch),			\
+			.vbp = DT_INST_PROP(n, vback_porch),			\
```

Comment:
can be done later, but we should likely have a common struct+initializer so that each driver doesn't need to copy this...